src/en/en.json

```diff
@@ -15,6 +15,7 @@
     "alert": "/en/coming-soon",
     "breadcrumbs": "/en/components/breadcrumbs",
     "button": "/en/components/button",
+    "card": "/en/coming-soon",
```

Comment:
Can you change this to `"container": "/en/components/container",` to include the new container link?